modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/status/RequestedQuoteCommerceOrderStatusImpl.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.internal.order.status;
+
+import com.liferay.commerce.constants.CommerceOrderConstants;
+import com.liferay.commerce.model.CommerceOrder;
+import com.liferay.commerce.order.CommerceOrderValidatorRegistry;
+import com.liferay.commerce.order.status.CommerceOrderStatus;
+import com.liferay.commerce.payment.method.CommercePaymentMethodRegistry;
+import com.liferay.commerce.service.CommerceOrderLocalService;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.service.WorkflowDefinitionLinkLocalService;
+
+import java.util.Locale;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+
+/**
+ * @author Brian I. Kim
+ */
+@Component(
+	property = {
+		"commerce.order.status.key=" + RequestedQuoteCommerceOrderStatusImpl.KEY,
```

Comment:
Previously you used always the form QuoteRequested. Why here we have RequestedQuote? Is it possible to rename this?

Reply:
@smottal , I am using requestedQuote as the default pattern. I may have misnamed somewhere. Could you let me know if my understanding is wrong?

Reply:
@smottal , I see what you mean. I am using quote-requested specifically for the constant because that was the value the mock design used.

Reply:
@luca-pellizzon , what do you think about the renaming request? This one is trivial to change, but I still want it to pass BChan's review :pray:

Reply:
I think we should follow the same pattern of the status constant that is ORDER_STATUS_QUOTE_REQUESTED, so we'll use QuoteRequestedCommerceOrderStatusImpl.KEY